pollenniva/manifest.json

```diff
@@ -1,8 +1,8 @@
 {
     "domain": "pollenniva",
-    "name": "pollenniva",
+    "name": "Pollennivå",
     "documentation": "https://github.com/JohNan/home-assistant-pollenkoll",
     "dependencies": [],
-    "codeowners": [],
+    "codeowners": ["JohNan"],
```

Comment:
sorry to be nitpicking, but it should include the `@` also. `@JohNan`

Ref: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

Reply:
Sorry, i was a bit to fast and trouble of typing @ on my rdp session,, but i fixed it!